server/src/main/java/io/littlehorse/common/LHConfig.java

```diff
@@ -275,6 +283,17 @@ public static List<NewTopic> getAllTopics(
                 .configs(globalMetaCLConfig)
         );
 
+        // TODO: define the appropriate configuration for this topic -> For sure it shouldn't have compaction
```

Comment:
Is this the correct configuration for the metadata topic? 
@coltmcnealy-lh

Reply:
This is correct for the Metadata Command Topic.